provisioners/src/main/java/org/jboss/intersmash/provision/helm/wildfly/eap8/Eap8HelmChartReleaseAdapter.java

```diff
@@ -38,7 +38,7 @@
  * This adapter is compliant with the contract which is required by the
  * {@link org.jboss.intersmash.provision.helm.HelmChartOpenShiftProvisioner} logic, i.e. to
  * implement {@link HelmChartRelease}, and allows for us to leverage a generated
- * {@link Eap8HelmChartReleaseAdapter#adaptee}, i.e. in terms of UX, provide native release YAML definitions.
+ * {@link org.jboss.intersmash.provision.helm.wildfly.eap8.EapXp5HelmChartReleaseAdapter#adaptee}, i.e. in terms of UX, provide native release YAML definitions.
```

Comment:
Maybe this `{@link}` is not correct? It should relate to EAP 8 tooling, IIUC.

Reply:
Thanks, fixed.